### PR TITLE
Slight tweak to asset layout in digitised bag

### PR DESCRIPTION
**Pull request by tomcrane**

### CODE REVIEW

File: docs/rfcs/002-archival_storage/README.md [hunk 1]
```diff
@@ -142,18 +142,16 @@ Digitised content will be ingested using Goobi, which should provide the bag lay
 ```
 b22036593/
 |-- data
-|   |-- b22036593.xml      // mets file
-|  [|-- bb22036593_001.xml // multiple manifestations]
-|  [|-- bb22036593_002.xml // multiple manifestations]
+|   |-- b22036593.xml      // mets "anchor" file for multiple manifestation
+|  [|-- bb22036593_001.xml // mets file for vol 1]
```
Comment (jtweed) [inline docs/rfcs/002-archival_storage/README.md]: Can you fix my bb typo while you're in here - thank you!

File: docs/rfcs/002-archival_storage/README.md [hunk 2]
```diff
@@ -142,18 +142,16 @@ Digitised content will be ingested using Goobi, which should provide the bag lay
 ```
 b22036593/
 |-- data
-|   |-- b22036593.xml      // mets file
-|  [|-- bb22036593_001.xml // multiple manifestations]
-|  [|-- bb22036593_002.xml // multiple manifestations]
+|   |-- b22036593.xml      // mets "anchor" file for multiple manifestation
```
Comment (jtweed) [inline docs/rfcs/002-archival_storage/README.md]: Also the only METS file where it's not a MM...

File: docs/rfcs/002-archival_storage/README.md (hunk 1 above)
Comment (tomcrane) [reply docs/rfcs/002-archival_storage/README.md]: done

File: docs/rfcs/002-archival_storage/README.md (hunk 2 above)
Comment (tomcrane) [reply docs/rfcs/002-archival_storage/README.md]: changed the b number to a real MM

**Comment by tomcrane** [comment]
Made clear that the JP2s and ALTOs are all in the same directory in the bag, even for MMs.

**Pull request merged**